docs/python101/python101-spec.tex

```diff
@@ -120,7 +120,7 @@ \section{Type system}
 
 $\infer[\text{(and)}]%
 {\Delta;\Gamma \vdash \kw{and}~\textrm{e}_\textrm{1},~\textrm{e}_\textrm{2} \yield \kw{bool}}%
-{\Delta;\Gamma \vdash \textrm{e}_\textrm{1} \colcol \kw{bool}~~ \Delta;\Gamma \vdash\textrm{e}_\textrm{2} \colcol \kw{bool} & \kw{and}:\kw{bool}\rightarrow\kw{bool}\rightarrow\kw{bool}\in \Delta}$
+{\Delta;\Gamma \vdash \textrm{e}_\textrm{1} \colcol \kw{bool}~~ \Delta;\Gamma \vdash\textrm{e}_\textrm{2} \colcol \kw{bool} & \kw{and}:\kw{bool}\times\kw{bool}\rightarrow\kw{bool}\in \Delta}$
```

Comment:
Be careful, we are not in a lambda-calculus, the arguments are a cartesian product, Python functions are not currified so it's an \rightarrow.

Reply:
so now it's good ;-)